packages/ecommerce-backend/render.yaml

```
@@ -0,0 +1,12 @@
+services:
+  # Deploy backend for the ecommerce
+  - type: web
+    name: ecomerceBackend
+    runtime: node
+    region: oregon
+    plan: standard
```

Debería ser free, ¿no?
```suggestion
    plan: free
```

---

you are right, done